bwameth.py

```diff
@@ -84,7 +109,7 @@ def convert_reads(fq1s, fq2s, out=sys.stdout):
         else:
             already_interleaved = False
 
-        q1_iter = izip(*[fq1] * 4)
+        q1_iter = izip(*[chain(first_five,fq1)] * 4)
```

Comment:
can you use chain.from_iterable as in a6a2ea6842939 so that it's not read into memory (this is fixing my previous bug).